src/shogun/base/SGObject.cpp

```diff
@@ -741,3 +754,14 @@ CSGObject* CSGObject::clone()
 	SG_DEBUG("leaving %s::clone(): Clone successful\n", get_name());
 	return copy;
 }
+
+void CSGObject::set(int tag_id, const Any& any)
```

Comment:
@lisitsyn and they are defined in the header file [here](https://github.com/shogun-toolbox/shogun/pull/3221/files#diff-1689b0bd3ceb28355cc3c05ed7ffe4f0R362).